packages/create-pantheon-decoupled-kit/src/templates/partials/next-drupal/nextDrupalPkgJson.hbs

```diff
@@ -0,0 +1,48 @@
+{
+	{{> sharedPkgJsonFields}}
+	"scripts": {
+		"dev": "next dev",
+		"build": "next build && cp -r .next/static .next/standalone/.next && cp -r public .next/standalone",
+		"start": "node .next/standalone/server.js",
+		"build:mono": "next build",
+		"start:mono": "next start",
+		"lint": "next lint",
+		"test": "vitest run",
+		"update-snapshots": "vitest run --update --silent",
+        {{#if tailwindcss}}
+        {{#if convertCssModules}}
+        "translate-vars": "postcss components/*.css pages/**/*.css --replace",
+		"convert-to-tailwind": "css-modules-to-tailwind pages/**/*.jsx  components/*.jsx --force",
```

Comment:
I'm not sure about this approach. Ideally, we would not need to leave extra scripts in the output. I would expect the action to run these commands on the output instead. That way we could keep all of the dependencies in the CLI instead of adding one-time use packages to the user's output.

I think the switch is not so bad – add the dependencies to the CLI and call `npx postcss components/*.css pages/**/*.css --replace` and `npx css-modules-to-tailwind pages/**/*.jsx components/*.jsx --force` using `execSync` like we're doing for these scripts.